sunpy/map/compositemap.py

```diff
@@ -466,7 +470,11 @@ def plot(self, axes=None, annotate=True,
                 ret.append(m.draw_contours(m.levels, **params))
 
                 # Set the label of the first line so a legend can be created
-                ret[-1].collections[0].set_label(m.name)
+                # TODO: remove when we depend on matplotlib 3.8 or later
```

Comment:
Let's try to keep track of this, maybe via an issue or something.

Reply:
I will open an issue!